src/reference/asciidoc/kafka.adoc

```diff
@@ -2400,12 +2400,13 @@ DefaultKafkaConsumerFactory<Integer, Cat1> cf = new DefaultKafkaConsumerFactory<
 ----
 ====
 
+[[messaging-message-conversion]]
 ===== Spring Messaging Message Conversion
 
 Although the `Serializer` and `Deserializer` API is quite simple and flexible from the low-level Kafka `Consumer` and
```

Comment:
Can we fix these *one sentence per line* violations as well in this PR?

Thanks